paths/translations/create.yaml

```
@@ -54,7 +54,7 @@ requestBody:
             type: string
             example: my-feature-branch
           locale_id:
-            description: Locale. Can be the name or public id of the locale. Preferred is the public id.
+            description: Locale. Can be the name or id of the locale. Preferred is the id
```

Just a nit (absolutely not something that would block the merging).
Just think article `the` is not needed here.

```suggestion
            description: Locale. Can be the name or id of the locale. Preferred is id
```